app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/Save.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action;
+use Magento\Framework\App\Request\DataPersistorInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class Save
+ * @package BodenkoVV\AskQuestion\Controller\Adminhtml\Question
+ */
+class Save extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /**
+     * @var PostDataProcessor
+     */
+    protected $dataProcessor;
+
+    /**
+     * @var DataPersistorInterface
+     */
+    protected $dataPersistor;
+
+    /**
+     * @var \Magento\Cms\Model\PageFactory
+     */
+    private $pageFactory;
+
+    /**
+     * @var \Magento\Cms\Api\PageRepositoryInterface
+     */
+    private $pageRepository;
+
+    /**
+     * @param Action\Context $context
+     * @param PostDataProcessor $dataProcessor
+     * @param DataPersistorInterface $dataPersistor
+     * @param \BodenkoVV\AskQuestion\Model\QuestionFactory $pageFactory
+     */
+    public function __construct(
+        Action\Context $context,
+        PostDataProcessor $dataProcessor,
+        DataPersistorInterface $dataPersistor,
+        \BodenkoVV\AskQuestion\Model\QuestionFactory $pageFactory = null
+    ) {
+        $this->dataProcessor = $dataProcessor;
+        $this->dataPersistor = $dataPersistor;
+        $this->pageFactory = $pageFactory
```

Comment:
Rename to yourfactory (questionFactory I mean)

Reply:
+